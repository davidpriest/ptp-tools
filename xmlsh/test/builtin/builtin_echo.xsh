# Tests of builtin echo 
#
# echo a blank line
echo 
# echo a string
echo "hi there"
# echo an XML expression 
echo <[<foo>bar</foo>]>
# echo an xml sequence seperated by space
echo <[1,"hi",<foo/>]>
# echo with variable expansion
A=var1
B=var2

# echo with null middle variable
echo $A$C$B
# echo quoted
echo "$A$C$B"
#
# echo single quoted
echo '$A$C$B'

# Echo -n
echo -n foo ; echo bar

# Echo to a named port
echo -p output to output
echo -p xxx to xxx (xxx)>&(output)






