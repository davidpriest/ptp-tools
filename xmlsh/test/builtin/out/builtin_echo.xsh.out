
hi there
<foo>bar</foo>
1 hi <foo/>
var1var2
var1var2
$A$C$B
foobar
to output
to xxx
